packages/itwin/map-layers/src/ui/FeatureInfoUiItemsProvider.tsx

```diff
@@ -4,53 +4,50 @@
 
 export class FeatureInfoUiItemsProvider implements UiItemsProvider { // eslint-disable-line deprecation/deprecation
   public readonly id = "FeatureInfoUiItemsProvider";
   public static readonly widgetId = "map-layers:mapFeatureInfoWidget";
 
   public constructor(private _featureInfoOpts: MapFeatureInfoOptions) { }
 
-  // public provideToolbarButtonItems(
-  //   _stageId: string,
-  //   stageUsage: string,
-  //   toolbarUsage: ToolbarUsage, // eslint-disable-line deprecation/deprecation
-  //   toolbarOrientation: ToolbarOrientation, // eslint-disable-line deprecation/deprecation
-  // ): CommonToolbarItem[] {
-  //   if (
-  //     !this._featureInfoOpts?.disableDefaultFeatureInfoTool &&
-  //     stageUsage === StageUsage.General && // eslint-disable-line deprecation/deprecation
-  //     toolbarUsage === ToolbarUsage.ContentManipulation && // eslint-disable-line deprecation/deprecation
-  //     toolbarOrientation === ToolbarOrientation.Vertical // eslint-disable-line deprecation/deprecation
-  //   ) {
-  //     DefaultMapFeatureInfoTool.register(MapLayersUI.localizationNamespace);
-  //     return [
-  //       ToolbarHelper.createToolbarItemFromItemDef(60, getDefaultMapFeatureInfoToolItemDef()),
-  //     ];
-  //   }
-
-  //   return [];
-  // }
-
-  // eslint-disable-next-line deprecation/deprecation
-  public provideWidgets(_stageId: string, stageUsage: string, location: StagePanelLocation, section?: StagePanelSection, zoneLocation?: AbstractZoneLocation): ReadonlyArray<AbstractWidgetProps> {
-    const widgets: AbstractWidgetProps[] = []; // eslint-disable-line deprecation/deprecation
-
-    // eslint-disable-next-line deprecation/deprecation
-    if ((undefined === section && stageUsage === StageUsage.General && zoneLocation === AbstractZoneLocation.BottomRight) ||
-      (stageUsage === StageUsage.General && location === StagePanelLocation.Right && section === StagePanelSection.End )) { // eslint-disable-line deprecation/deprecation
+  public provideToolbarItems(
+    _stageId: string,
+    stageUsage: string,
+    toolbarUsage: ToolbarUsage,
+    toolbarOrientation: ToolbarOrientation,
+  ): ReadonlyArray<ToolbarItem> {
+    if (
+      !this._featureInfoOpts?.disableDefaultFeatureInfoTool &&
+      stageUsage === StageUsage.General &&
+      toolbarUsage === ToolbarUsage.ContentManipulation &&
+      toolbarOrientation === ToolbarOrientation.Vertical
+    ) {
+      DefaultMapFeatureInfoTool.register(MapLayersUI.localizationNamespace);
+      return [
+        ToolbarHelper.createToolbarItemFromItemDef(60, getDefaultMapFeatureInfoToolItemDef()),
+      ];
+    }
+
+    return [];
+  }
+
+  public provideWidgets(_stageId: string, stageUsage: string, location: StagePanelLocation, section?: StagePanelSection) {
+    const widgets = [];
+
+    if ((undefined === section && stageUsage === StageUsage.General && location === StagePanelLocation.Right ) ||
```

Comment:
i think this can be condensed. previously had multiple conditions to support ui1.0 which is killed off in appui4.0

Reply:
should be better now.